src/util/authUtils.test.js

```diff
@@ -0,0 +1,36 @@
+import {
+	// applyAuthState,
```

Comment:
yeah, these need tests, so it's sort of a TODO but I didn't want to junk up this branch with unrelated tests - would you prefer the commented lines be removed or is it alright to leave them here as an indication that there are untested functions in authUtils. I'll go ahead and make a follow-up task